main.go

```diff
@@ -60,11 +61,16 @@ func restoreNotifications(path string) map[string]map[string]Notification {
 
 		var decodedData map[string]map[string]Notification
 		err = dec.Decode(&decodedData)
+		if err != nil {
+			log.Printf("Error could not decode %v", dec)
+			goto err
+		}
+		log.Printf("Restored notifications %v", decodedData)
 		return decodedData
 	}
 	log.Println("No notification storage found, not restoring anything")
-	err:
-		return make(map[string]map[string]Notification)
+err:
+	return make(map[string]map[string]Notification)
 }
 
 func saveNotifications(notifs map[string]map[string]Notification, path string) error {
```

Comment:
And same for the `restoreNotifications` => `func (n Notifications) restore(path string)`